src/Core/Silk.NET.SilkTouch/NativeContextOverrideHelper.cs

```diff
@@ -0,0 +1,332 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Silk.NET.SilkTouch
+{
+    public static class NativeContextOverrideHelper
+    {
+        public static MethodDeclarationSyntax GetProcAddress => HelperMethods.Value.GetProcAddress;
```

Comment:
this lazy seems entirely pointless. SyntaxFactory isn't very expensive and this code is impossible to read (in part) because of the lazy + tuple combination.
Additionally, I'd prefer if this was cleaned up again, roslyn quoter does a great job, but often isn't very smart about constructor use, and this massively bloats the code size in cases like this.

Reply:
> this lazy seems entirely pointless. SyntaxFactory isn't very expensive and this code is impossible to read (in part) because of the lazy + tuple combination.

Perhaps not performance-wise, but it's a lot of unnecessary memory stress otherwise.

> Additionally, I'd prefer if this was cleaned up again, roslyn quoter does a great job, but often isn't very smart about constructor use, and this massively bloats the code size in cases like this.

Well I don't know what's unnecessary and what's not, this isn't really my domain.

Reply:
Fair enough. I'll clean it up afterwards; if you'd like to give it a shot, it's pretty much just "look what you can give the constructor and give it that" x1000 - but yeah, probably necessary to pick that up.